.github/workflows/regression-report.yml

```diff
@@ -122,11 +126,172 @@ jobs:
           # shellcheck disable=SC2005
           echo "$(cat "$GITHUB_OUTPUT")"
 
-  make_report:
+
+  handle_config_bundle:
+    needs: parse_comment
+    if: needs.parse_comment.outputs.mode == 'use_config_bundle'
     runs-on: ubuntu-latest
+    outputs:
+      config_link: ${{ steps.set_links.outputs.config_link }}
+      projects_link: ${{ steps.set_links.outputs.projects_link }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
```

Comment:
checkout of code is network operation, it is fragile conceptually.
can we checkout code 2 time max: for PR and for master branch.

and store both for other jobs in cache of action (not an artifacts).

Reply:
done